components/automate-ui/src/app/entities/layout/layout-sidebar.service.ts

```diff
@@ -271,10 +271,10 @@ export class LayoutSidebarService {
 
     public getUserProfileSidebar(): MenuItemGroup[] {
         return [{
-            name: '',
+            name: 'User Menu',
```

Comment:
Updating the left sidebar to look like below.
![image](https://user-images.githubusercontent.com/1679247/72574413-ce370080-387d-11ea-839a-c1c5961a9b13.png)